### ISSUE TITLE
Reach out to new communities

### ISSUE THREAD

**Issue by NatalieThurlby** [very-good-science/data-ethics-club#85]
I'd like to make sure that Data Ethics Club has a diversity of points of view and experience, and while I think we hit that really well in terms of disciplines within a university research and data science context, I'm wondering how we could reach out to new communities, particularly:
- interested members of the public 
- small tech companies
- analysts in NHS/police/local + central government roles
- undergraduate/masters students

In addition, across all these categories and our existing community of data scientists and researchers, I also wonder what we can do to make Data Ethics Club more appealing and welcoming for a more diverse demographic of people. So, part of that might be advertising, so within the university, maybe we could reach out to the BAME network, or if we did a "decolonising data science" meeting, then the decolonise UoB group might be interested?

I'm leaving this issue here to gather some ideas for specific groups we could reach out to, or perhaps a special edition of Data Ethics Club that is in real life, invites the public, maybe at the JGI showcase (which will be in February at the MShed) for example. Then hopefully we can enact something from here at our next organiser's meeting.

**Comment by ninadicara** [comment]
Yes, I really like the idea of doing some thinking on who we could reach out to! I do know someone who has done work for their PhD on talking to the public about AI who I could reach out to and ask for their thoughts. Another option might be to reach out to We The Curious and ask if they might be interested in helping us host a Data Ethics Club there - it could even be child/family focussed.

**Comment by NatalieThurlby** [comment]
Oooh, Nina I LOVE the science museum idea!
